src/Agent.Worker/Release/AgentUtilities.cs

```diff
@@ -13,7 +14,8 @@ public static string GetPrintableEnvironmentVariables(IEnumerable<KeyValuePair<s
 
             if (variables != null)
             {
-                foreach (var pair in variables)
+                var sortedVariables = variables.OrderBy(x => x.Key, StringComparer.OrdinalIgnoreCase);
```

Comment:
Hi Madhu, 
We are using this function from only one place and so we will always be sorting it for now. If someone requires the variables in same order we can refactor the function then.

Also, I think it is easier to sort the whole enumerable at once rather than filtering the keys and then only sorting the keys, looks cleaner to me.